src/streamlink/plugin/plugin.py

```diff
@@ -254,7 +255,7 @@ def streams(self, stream_types=None, sorting_excludes=None):
                                 key=partial(stream_type_priority,
                                             stream_types))
 
-        streams = {}
+        streams = OrderedDict()
```

Comment:
I missed that.  I originally relied on a slightly different technique on my original attempt to sort, and this is a remnant of that.